config/src/lib.rs

```diff
@@ -1,11 +1,296 @@
-pub fn hello_world() {
-    println!("Hello World!!");
+use std::ffi::OsString;
```

Comment:
Ideally you'd also add a short crate/module level comment at the top

It's lines would need to begin with "//!" instead of "///"

Reply:
Done